doc/container-environment.md

```diff
@@ -62,7 +62,7 @@ The following mounts are set up by default:
 
 If they are present on the host, the following paths will also automatically be mounted:
 
-- `/proc/sys/fs/binfmt_misc`
+- `/proc/sys/fs/binfmt_misc` (only on kernels that lack binfmt_misc namespace support)
```

Comment:
If that `binfmt_misc` bit inside the parenthesis is what prompted you to add it to the custom word list, I think it'd be better to just surround it with backticks which is enough to get the spellchecker to stop looking for typos.